### ISSUE TITLE
Favorites don't show the LMS image

### ISSUE THREAD

**Issue by Haifeif** [oweitman/ioBroker.squeezeboxrpc#102]
The favorites widget doesn't show the in LMS stored image, but the LMS standard, "html/images/favorites.png". Also, if there is no defined image in LMS, the widget doesn't show the favorites name.

**Expected behavior** 
Take the LMS image, without stretching. Let us set xy dimensions for the images ourselfs, 1 setup for all
If there is no image in LMS or the favorites.png, show no image and unstead the Favorites name.

**Versions:**  
Adapter version: 1.3.17
JS-Controller version: 6.0.11
Node version: v20.17.0
Operating system: LINUX

I hope I don't annoy you with my reports. But delivered the basic for project, I'm working on now for a year. All other adapters, skripts and you name it were crap. Your adapter is exactly, what I need. So I hope you see me as an Beta Tester for a great piece of software!

**Comment by oweitman** [comment]
Ja, an dieser Stelle hatte ich etwas angepasst und auch noch ein paar Lücken gefüllt, die bisher noch nicht berücksichtigt waren.
Also
- du hast im LMS ein Favorit definiert? 
- Welcher Typ ist das? Ein Song, ein Stream?
- Der LMS zeigt in der Favoritenauflistung als kleines Icon auch dieses Bild an, oder auch dort nur das favorites dummy bild?
- Hat das schon einmal funktioniert?

**Comment by oweitman** [comment]
ok, also für gespeicherte titels, bei denen ein cover gepseichert ist,
wurde das bild ja schon geliefert
für gestreamte sender, gab es diese bilder eigentlich nie, da in den informationen im stream nicht enthalten ist.
wenn man den sender in den favortien gespeichert hat, dann wird ein dummy bild angezeigt (herz) oder manchmal auch dummy bild sender, woran der lms das unterscheidet weiß ich nicht.
für die favortien kann man manuell in die datei in der die daten gespeichert sind selbst ein bild hinzufügen.
datei nennt sich /prefs/favorites.opml
die eigenschaft die man ergänzen muss nennt sich icon oder image (weiß grad nicht mehr welches von beiden das richtigere ist, ich hab im zweifel beide gepflegt)
die datei darf man aber nur bearbeiten, wenn der lms nicht läuft. sonst sind die änderungen wieder weg

für das image widget hab ich eine Ergänzung gemacht, (die dann in der kommenden alpha2 enthalten ist), das die dummy bilder nun angezeigt werden.
bisher hat das wohl niemand gestört, daher hab ich das nie eingefügt.
